docs/source/guides/dfs_with_dask_entitysets.rst

```diff
@@ -0,0 +1,102 @@
+
+.. ipython:: python
+
+    feature_matrix, features = ft.dfs(entityset=es,
+                                      target_entity="dask_entity",
+                                      trans_primitives=["negate"])
+    feature_matrix
+
+
+This feature matrix can be saved to disk or computed and brought into memory, using the appropriate Dask dataframe methods.
+
+.. ipython:: python
+
+    fm_computed = feature_matrix.compute()
+    fm_computed
+
+
+While this is a simple example to illustrate the process of using Dask dataframes with Featuretools, this process will also work with entitysets containing multiple entities, as well as with aggregation primitives.
+
+Limitations
+-----------
+There are many parts of Featuretools that are difficult to implement in a distributed environment and several primitives that are not well suited to operate on distributed dataframes. As a result, there are some limitations when creating Dask entitysets and then using these entitysets to generate a feature matrix. The most significant limitations are reviewed in more detail in this section.
+
+Supported Primitives
+********************
+When creating a feature matrix from a Dask entityset, only certain primitives can be used. Computation of certain features is quite expensive in a distributed environment, and as a result only a subset of Featuretools primitives are currently supported when using a Dask entityset.
+
+To obtain a list of the primitives that can be used with Dask entitysets, you can call ``featuretools.list_primitives()``. This will return a table of all primitives. Any primitive that can be used with a Dask entityset will have a value of ``True`` in the ``dask_compatible`` column.
+
+
+.. ipython:: python
+
+    primitives_df = ft.list_primitives()
+    dask_compatible_df = primitives_df[primitives_df["dask_compatible"] == True]
+    dask_compatible_df.head()
+    dask_compatible_df.tail()
+
+
+Entityset Limitations
+*********************
+When creating a Featuretools ``Entity`` from Dask dataframes, variable type inference is not performed as it is when creating entities from pandas dataframes. This is done to improve speed as sampling the data to infer the variable types would require an expensive compute operation on the underlying Dask dataframe. As a consequence of, this users must define the variable types for each column in the supplied Dataframe. This step is needed so that the deep feature synthesis process can build the proper features based on the column types. A list of available variable types can be obtained by running ``featuretools.variable_types.find_variable_types()``.
+
+By default, Featuretools checks that entities created from pandas dataframes have unique index values. Because performing this same check with Dask would require an expensive compute operation, this check is not performed when creating an entity from a Dask dataframe. When using Dask dataframes, users must ensure that the supplied index values are unique.
```

Comment:
We should add a section to point out that unlike with Pandas, a distributed dataframe's order is not guaranteed and we currently don't do anything to try to maintain an order